presto-hive-hadoop2/src/test/java/io/prestosql/plugin/hive/TestHiveHadoop2Plugin.java

```diff
@@ -38,6 +38,7 @@
 import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
squash this commit please with previous one.

Reply:
which one?

Reply:
It seems that it is squashed now.